### ISSUE TITLE
[Proposal] Move the pos classes to math package

### ISSUE THREAD

**Issue by SettingDust** [SilkMC/silk#43]
I think them can be used in rendering and so on. The conversion for chunkpos and blockpos should be extensions in world package

**Comment by jakobkmar** [comment]
Yes I thought about moving them as well - but when moving them we should also discuss whether `Pos**` is even the correct naming scheme. Maybe we should just call then `Vec**`, but then they are not that well separated from the Mojang vectors. @F0Xde what do you think?
